index.html

```diff
@@ -0,0 +1,133 @@
+    </div>
+
+    <div class="main--flex-images-container">
+        <div class="main--flex-div">
+            <img class="main--flex-img boatImg" src="images/banners/boatBanner.jpg" alt="">
+        </div>
+
+        <div class="main--flex-div">
+            <img class="main--flex-img heartHandsImg" src="images/banners/heartHandsBanner.jpg" alt="">
+        </div>
+
+        <div class="main--flex-div ">
+            <img class="main--flex-img trackingImg" src="images/banners/trackingBanner.jpg" alt="">
+        </div>
+
+        <div class="main--flex-div ">
+            <img class="main--flex-img workerImg" src="images/banners/workerBanner.jpg" alt="">
+        </div>
+    </div>
+
+    <div class="main--feedback-container">
+        <img class="main--vaultariBackground-img" src="images/banners/vaultariBackground.jpg" alt="">
+    </div>
+
+    <div class="main--description-container">
+        <img class="main--description-img" src="images/banners/descriptionBackground.jpg" alt="">
+    </div>
+
+    <div class="main--submitQuote-container">
+        <img class="main--laptop-img" src="images/banners/laptopBackground.jpg" alt="">
+
+        <button class="main--submitQuote-button button">GET FREE QUOTE</button>
+    </div>
+
+    <div class="main--ourPartners-container">
+        <img class="main--partnersBackground-img" src="images/banners/ourPartnersBackground.jpg" alt="">
+    </div>
+
+    <div class="main--contact-container">
+
+        <div class="main--contact-images-container">
+            <a class="contact-logo-img-grid" href=""><img class="main--contact-logo-img" src="images/logos/logo copy.png" alt=""></a>
+            <a class="social1-contact" href=""><img class="main--contact-facebook-img" src="images/logos/facebook-logo.png" alt=""></a>
+            <a class="social2-contact" href=""><img class="main--contact-linkedIn-img" src="images/logos/twitter-logo.png" alt=""></a>
+            <a class="social3-contact" href=""><img class="main--contact-twitter-im " src="images/logos/linkedin-logo.png" alt=""></a>
+        </div>
+
+        <div class="main--contact-flex-products-container position">
+            <span>products</span>
+            <ul>
+                <a href=""><li>commercial lines</li></a>
+                <a href=""><li>non profit insurance</li></a>
+                <a href=""><li>speciality industry & programs</li></a>
+                <a href=""><li>coverage types & descriptios</li></a>
+            </ul>
+        </div>
+
+        <div class="main--contact-flex-contactInfo-container position">
+            <span>contact info</span>
```

Comment:
interesant ca mai sus ai pus frumos `H1` mai sus .. dar asta l ai lasat `span` .. eu cumva le vad ca niste titluri mai mici .. as fi mers pe `H2` ... dar sunt curios cum vezi tu lucrurile .. cand fol `H1-H6` si cand fol `span`

Reply:
in psd nu trebuia sa aiba spatiul asa mare, cum pune by default h2, de-aia dar ma reorientez